drivers/usb/device/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
-zephyr_sources_ifdef(CONFIG_USB_DW       usb_dc_dw.c)
-zephyr_sources_ifdef(CONFIG_USB_DC_STM32 usb_dc_stm32.c)
-zephyr_sources_ifdef(CONFIG_USB_DC_SAM0  usb_dc_sam0.c)
-zephyr_sources_ifdef(CONFIG_USB_DC_SAM   usb_dc_sam.c)
-zephyr_sources_ifdef(CONFIG_USB_NRFX     usb_dc_nrfx.c)
-zephyr_sources_ifdef(CONFIG_USB_KINETIS  usb_dc_kinetis.c)
+zephyr_sources_ifdef(CONFIG_USB_DW           usb_dc_dw.c)
+zephyr_sources_ifdef(CONFIG_USB_DC_STM32     usb_dc_stm32.c)
+zephyr_sources_ifdef(CONFIG_USB_DC_SAM0      usb_dc_sam0.c)
+zephyr_sources_ifdef(CONFIG_USB_DC_SAM_USBHS usb_dc_sam_usbhs.c)
```

Comment:
Please move this patch to own PR, this does not draft/belong here. Also the new driver does not seem to be in conflict with usb_dc_sam.c driver.

Reply:
Done #36763